force-app/main/default/classes/MonthlyBillController.cls

```diff
@@ -25,7 +25,7 @@ public with sharing class MonthlyBillController {
           AND Month__c = :metersReading.Month__c
           AND Year__c = :metersReading.Year__c
         LIMIT 1
-      ][0];
+      ];
```

Comment:
Lines 6-28: It's always better to merge the two queries into one when possible. Check if you can use a subquery here.

Reply:
We discussed it before, I don't know if it is possible in such db schema:
![123](https://user-images.githubusercontent.com/60067042/203307378-2606bbd3-c14c-41a2-a403-f3392f72415a.png)